django/contrib/admin/static/admin/css/forms.css

```diff
@@ -158,17 +171,22 @@ fieldset.collapsed .collapse-toggle {
 /* MONOSPACE TEXTAREAS */
 
 fieldset.monospace textarea {
-    font-family: "Bitstream Vera Sans Mono",Monaco,"Courier New",Courier,monospace;
+    font-family: "Roboto", Corbel, Avenir, "Lucida Grande", "Lucida Sans", sans-serif;
+}
+
+textarea {
+    resize: vertical;
 }
 
 /* SUBMIT ROW */
 
 .submit-row {
-    padding: 5px 7px;
+    padding: 20px;
+    margin: 40px 0 0;
```

Comment:
This 40px margin leads to a lot of extra header whitespace if `save_on_top = True`. Could it be reduced?